src/oca_github_bot/tasks/tag_needs_review.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) ACSONE SA/NV 2018
```

Comment:
Are you sure?

Reply:
LOL, used another file as base for this and forgot to remove this